rpc/client.ts

```diff
@@ -47,7 +47,7 @@ export class Client<
         const pendingCall = this.pendingCalls[id]!
         pendingCall.resolve(e)
         delete this.pendingCalls[id]
-        this.pendingSubscriptions[id]!(e)
+        this.pendingSubscriptions[id]?.(e)
```

Comment:
bug fix, on error there may not be pendingSubscriptions for the given message id

Reply:
This is a *very* important fix, as these errors were not getting caught, and thus crashed the process, including on the codegen server.